pkg/chartutil/load.go

```diff
@@ -48,6 +48,9 @@ func Load(name string) (*chart.Chart, error) {
 		return nil, err
 	}
 	if fi.IsDir() {
+		if validChart, err := IsChartDir(name); !validChart {
```

Comment:
hmm, I checked this, and the checks I have added in IsChartDir are already pre-checked while calling linter at:
a) https://github.com/kubernetes/helm/blob/master/cmd/helm/lint.go#L145-L147

b) https://github.com/kubernetes/helm/blob/master/pkg/lint/lint.go#L32